scripts/dist-management/download-dist.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 const fs = require('fs');
 
-const request = require('request');
+const fetch = require('node-fetch');
```

Comment:
@molant do you know what this script is used for?

Reply:
IIRC, this is the script used to download the generated build artifacts when doing a fresh cloned. This should speed up things under certain circumstances, especially when cloning the project.